src/util/config.rs

```diff
@@ -739,6 +739,132 @@ pub fn check_kernel() -> Vec<ConfigError> {
     Vec::new()
 }
 
+#[derive(Debug, Default)]
```

Comment:
can we move this to another file, it is crowded in the config file now. 

Maybe we should create a config directory now.

Reply:
Maybe we could refactor the config file into a directory in another PR?